### PR TITLE
Updated ci-build-image-updates docs

### PR DESCRIPTION
**Pull request by devkimittal**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by anurag-harness — APPROVED** [review]

**Review by rajatharanganath — APPROVED** [review]

**Review by hemanthmantri — APPROVED** [review]

File: docs/continuous-integration/ci-technical-reference/ci-build-image-updates.md [hunk 1]
```diff
@@ -60,30 +59,30 @@ The response payload shows the latest supported images and their tags.
...
+    }
 ]'
 ```
 4. To reset one or more images to their defaults, send a`reset-config` (POST) request with a list of the images to reset.
 ```
-curl --location --request POST 'https://app.harness.io/gateway/ci/execution-config/reset-config?accountIdentifier=$ACCOUNT_ID&infra=K8' \  
---header 'Authorization: Bearer $API_KEY' \  
---header 'Content-Type: application/json' \  
---data-raw '[  
+curl --location --request POST "https://app.harness.io/gateway/ci/execution-config/reset-config?accountIdentifier=$ACCOUNT_ID&infra=K8" --header 'Authorization: Bearer $API_KEY'  --header 'Content-Type: application/json'  --data-raw '[
```
Comment (aimurphy) [inline docs/continuous-integration/ci-technical-reference/ci-build-image-updates.md:80]: @devkimittal Was there a specific reason for removing the line continuation characters (\) in these code samples? They make the code more readable by not printing it all on one long line, which can be hard to read with horizontal scrolling.

Comment (devkimittal) [reply docs/continuous-integration/ci-technical-reference/ci-build-image-updates.md:80]: from my experience, customer is simply copying the command and running it on terminal, the command with / is not parsable in postman and gives some error on terminal.

Comment (devkimittal) [reply docs/continuous-integration/ci-technical-reference/ci-build-image-updates.md]: But I see your point, added new lines in between

Comment (aimurphy) [reply docs/continuous-integration/ci-technical-reference/ci-build-image-updates.md]: @devkimittal one line is fine. I didn't know Postman didn't accept the line continuation character.

Comment (devkimittal) [reply docs/continuous-integration/ci-technical-reference/ci-build-image-updates.md]: can we merge it now, the new view looks like this https://ci.pr.harness.io/pr-638/docs/continuous-integration/ci-technical-reference/ci-build-image-updates

**Review by aimurphy — APPROVED** [review]

**Pull request merged**